.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,29 +1,59 @@
-## Checklist
+### 🗒️ Checklist
 
-1. [ ] If you've added code that should be tested, add tests
-2. [ ] If you've changed APIs, update (or create!) the documentation
-3. [ ] Ensure the tests pass
-4. [ ] Run `./python-format.sh` to make sure that your code lints and that you've followed [our coding style](https://github.com/kobotoolbox/kpi/blob/main/CONTRIBUTING.md)
-5. [ ] Write a title and, if necessary, a description of your work suitable for publishing in our [release notes](https://community.kobotoolbox.org/tag/release-notes)
-6. [ ] Mention any related issues in this repository (as #ISSUE) and in other repositories (as kobotoolbox/other#ISSUE)
-7. [ ] Open an issue in the [docs](https://github.com/kobotoolbox/docs/issues/new) if there are UI/UX changes
-8. [ ] Create a testing plan for the reviewer and add it to the Testing section
-9. [ ] Add frontend or backend tag and any other appropriate tags to this pull request
+1. [ ] run linter locally
+2. [ ] update all related docs (API, README, inline, etc.), if any
+3. [ ] draft PR with a title `<type>(<scope>)<!>: <title> TASK-1234`
+4. [ ] tag PR: at least `frontend` or `backend` unless it's global
+5. [ ] fill in the template below and delete template comments
+6. [ ] review thyself: read the diff and repro the preview as written
+7. [ ] open PR & confirm that CI passes
+8. [ ] request reviewers, if needed
+9. [ ] delete this section once checklist is done
 
-## Description
+### 📣 Summary
+<!-- Delete this section if changes are internal only. -->
+<!-- One sentence summary for the public changelog, worded for non-technical seasoned Kobo users. -->
```

Comment:
@JacquelineMorrissette do you include in changelog internal only changes, like this PR?

Reply:
I include all PRs and I don't think we should change that because we are open source and there are individuals who maintain their own forks. This template, for example, would be changing for people who want to contribute too